example/lib/meeting/meeting_store.dart

```diff
@@ -1295,13 +1327,53 @@ class MeetingStore extends ChangeNotifier
   }
 
   void enterPipMode() async {
-    //to check whether pip is available
-    bool _isPipAvailable = await _hmsSDKInteractor.isPipAvailable();
-    if (_isPipAvailable) {
-      //[isPipActive] method can also be used to check whether application is in pip Mode or not
-      isPipActive =
-          await _hmsSDKInteractor.enterPipMode(autoEnterPip: isPipAutoEnabled);
-      notifyListeners();
+    //to check whether pip is available in android
+    if (Platform.isAndroid) {
+      bool _isPipAvailable = await HMSAndroidPIPController.isAvailable();
+      if (_isPipAvailable) {
+        //[isPipActive] method can also be used to check whether application is in pip Mode or not
+        isPipActive =
+            await HMSAndroidPIPController.start(autoEnterPip: isPipAutoEnabled);
+        notifyListeners();
+      }
+    }
+  }
+
+  Future<bool> isPIPActive() async {
+    if (Platform.isAndroid)
+      isPipActive = await HMSAndroidPIPController.isActive();
+    else if (Platform.isIOS) isPipActive = await HMSIOSPIPController.isActive();
+    return isPipActive;
+  }
+
+  void changeTrackPIP(
+      {HMSVideoTrack? track,
+      required String alternativeText,
+      required List<int> ratio}) async {
+    if (Platform.isIOS && track != null) {
+      isPipActive = await isPIPActive();
+      if (isPipActive) {
+        HMSIOSPIPController.changeTrack(
+            track: track,
+            aspectRatio: ratio,
+            alternativeText: alternativeText,
+            scaleType: ScaleType.SCALE_ASPECT_FILL);
+      }
+    }
+  }
+
+  void changeTextPIP({String? text, required List<int> ratio}) async {
+    if (Platform.isIOS && text != null) {
+      isPipActive = await isPIPActive();
+      if (isPipActive) {
+        HMSIOSPIPController.changeText(text: text, aspectRatio: ratio);
+      }
+    }
+  }
+
+  void destroyPIP() {
```

Comment:
rename function to avoid Android / iOS confusion for readers
```suggestion
  void destroyPIPSetupOnIOS() {
```